src/android/com/eclipsesource/tabris/gcm/heartbeat/GCMHeartbeatHandler.kt

```diff
@@ -0,0 +1,15 @@
+package com.eclipsesource.tabris.gcm.heartbeat
+
+import com.eclipsesource.tabris.android.ActivityScope
+import com.eclipsesource.tabris.android.ObjectHandler
+import com.eclipsesource.v8.V8Object
+
+class GCMHeartbeatHandler(private val scope: ActivityScope)
+    : ObjectHandler<GCMHeartbeat> {
+    override val type = "com.eclipsesource.tabris.gcm.heartbeat.GCMHeartbeat"
+    override fun create(id: String, properties: V8Object) = GCMHeartbeat(scope)
+    override fun call(gcmHeartbeat: GCMHeartbeat, method: String, properties: V8Object) = when (method) {
+        "sendHeartbeat" -> gcmHeartbeat.sendHeartbeat()
+        else -> null
```

Comment:
The else case is not required here.

Reply:
The compiler says:
```
GCMHeartbeatHandler.kt: (14, 91): 'when' expression must be exhaustive, add necessary 'else' branch
```

@mpost Do you know what the issue might be?

Reply:
Ah ok. Than the else is required. If you use an `Enum` with all values used in the when, than it is not required.